mantidproject/mantid
# Crystal field theory docs

**Description of work.**



Adds a Crystal Field concepts page describing the theory behind the crystal field fit function calculations.

Also modified the links in the crystal fields techniques and interface page to reference each other.

**To test:**



Check that html documents (especially LaTeX / mathjax equations) build correctly.

Fixes #19498. 


*This does not require release notes* because it adds background documentation for already implemented functionality.



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
